shutdownAll

```diff
@@ -1,22 +1,25 @@
 #!/bin/bash
 
 ######################################################################
+#
+# Shut down all cluster nodes or a specific node
+#
 # Usage:
-#  shutdownAll
+#   ./shutdownAll [node-nr]
```

Comment:
should we call it just 'shutdown' then?

Reply:
Indeed: that’s what I was thinking about too. Will change.

Reply:
Just wondering... Renaming it to `shutdown` may be potentially dangerous/confusing as there's the possibility that a user executes `shutdown` instead of `./shutdown`. OTOH, the local `shutdown` command will require using sudo, so I guess there's no real risk. Will move ahead with the suggested change.